src/components/atoms/SideBarButton/SideBarButton.tsx

```diff
@@ -0,0 +1,16 @@
+import { SideBarButtonProps } from './ButtonInterface';
+
+const SideBarButton = ({ title, component: Icon }: SideBarButtonProps) => {
```

Comment:
components that are just pure jsx and dont have any logic can be written as [without return statement]:
```
import { SideBarButtonProps } from './ButtonInterface';

const SideBarButton = ({ title, component: Icon }: SideBarButtonProps) => (
    <div className="h-[25px] w-[170px]  flex items-center justify-between ">
      <div className="">
        <Icon height="25" width="25" color="#232323" />
      </div>
      <div className="w-3/4 h-[25px] bg-slate">
        <h1 className="font-medium text-[18px] ml-2">{title}</h1>
      </div>
    </div>
  );

export default SideBarButton;
```